src/main/java/CommandHandler.java

```diff
@@ -1,13 +1,11 @@
 //import jdk.internal.util.xml.impl.Input;
 
-import java.util.Vector;
+public class CommandHandler {
 
-public class commandHandler {
+    DisplayHandler DisplayUnit = new DisplayHandler();
+    InputParser InputUnit = new InputParser();
```

Comment:
According to the naming convention,  Variable names must be in camelCase. 
displayUnit, inputUnit and etc.

Reply:
Thanks for pointing this out. I missed this while refactoring.